roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -12,5 +12,12 @@
 - name: start aci-gw container
   service: name=aci-gw state=started
 
+- name: Contiv | Wait for netmaster
+  action: shell netctl tenant ls
+  register: tenant_result
+  until: tenant_result.stdout.find("default") != -1
+  retries: 9
+  delay: 10
+
 - name: set aci mode
-  shell: contivctl network global set --fabric-mode aci
+  shell: netctl global set --fabric-mode aci
```

Comment:
hmm, okay. Yes, we have had similar discussions earlier and for sometime now at least the idea was to show off a unified cli tool to the user which was the reason to use `contivctl` in ansible. And you are right that none of the projects internally use this tool (may be it is just a matter of updating docs).

I am okay using netctl here, if everyone else is on same page. And I would rather want to not have to maintain contivctl if no one is using it (i.e. remove it's installation from ansible).

ping to @jainvipin @shaleman @erikh as well. Hopefully we can close it this time :)